e2e-tests/tests/add-element.spec.ts

```diff
@@ -7,14 +7,14 @@ const expectedPage = fs.readFileSync(
   "utf-8"
 );
 
-studioTest("can add a Footer component", async ({ page, studioPage }) => {
+studioTest("can add a Footer component", async ({ studioPage }) => {
   await studioPage.switchPage("LocationPage");
   await studioPage.addElement("Footer", "Components");
   const previews = studioPage.preview.getByText("© 2023 Yext");
   await expect(previews).toHaveCount(1);
-  await expect(page).toHaveScreenshot();
+  await studioPage.takeScreenshotAfterImgRender();
```

Comment:
Should we make another item to go through and tighten the scopes on the screenshots, so that they're not all whole page captures?

If we do this, though, I think that we might need to add some more tests to make sure that we keep the same coverage. For example, we might need to add a test for the save button graying out after a save, depending on how we tighten these screenshots.